frontend/src/services/account/useAccount.ts

```diff
@@ -0,0 +1,44 @@
+import { useMutation, useQuery, useQueryClient } from '@tanstack/react-query'
+import { useContext } from 'react'
+import { DialogContext } from '@/context/DialogContext.ts'
+import { useUser } from '@/services/auth/useAuth.ts'
+import type { AccountRow } from '@/types'
+import { accountService } from './accountService'
+
+export function useAccount() {
+  const { data: user } = useUser()
+  const email = user?.user.email
+
+  return useQuery({
+    queryKey: ['account', email],
+    queryFn: () => accountService.fetchAccount(email || ''),
```

Comment:
```suggestion
    queryFn: () => accountService.fetchAccount(email as string),
```

I believe this was because of types, but it was already enforced in runtime by `enabled`, right? Is so, this cast should make it clear.